docker-compose.yml

```diff
@@ -108,6 +110,13 @@ services:
   redis-cache:
     image: redis:6.2.6
 
+  # https://github.com/fsouza/fake-gcs-server
+  # Fake GCP GCS server for local development and testing
+  gcs-emulator:
+    image: fsouza/fake-gcs-server:latest
+    ports:
+      - "${EXPOSE_GCS_EMULATOR_PORT:-5553}:4443"
```

Comment:
Strangely port 4443 for the host was already in use by VS Code Dev Containers once the `devcontainer` service was running in the `tecken` workspace, so without using a different host port, requests to the emulator service were just hanging.

Reply:
@relud ^^^ Can you weigh in on port usage?

Reply:
i have no answers, setting this to 4443 seems to be working for me 🤷

Reply:
setup:
```bash
git checkout bug-1827507
echo 'EXPOSE_GCS_EMULATOR_PORT=4443' >> .env
docker compose up gcs-emulator -d
```
how i confirmed it was working:
```bash
$ STORAGE_EMULATOR_HOST=https://localhost:4443 venv/bin/python bin/gcs_cli.py create relud
GCS bucket 'relud' in 'https://localhost:4443/storage/v1/b/relud' created.
$ STORAGE_EMULATOR_HOST=https://localhost:4443 venv/bin/python bin/gcs_cli.py delete relud
GCS bucket 'relud' at 'https://localhost:4443/storage/v1/b/relud' deleted.
```

this was with vscode and devcontainer running the whole time

Reply:
within my devcontainer i was able to access it with:
```bash
$ STORAGE_EMULATOR_HOST=https://gcs-emulator:4443 python bin/gcs_cli.py create relud
GCS bucket 'relud' in 'https://gcs-emulator:4443/storage/v1/b/relud' created.
$ STORAGE_EMULATOR_HOST=https://gcs-emulator:4443 python bin/gcs_cli.py delete relud
GCS bucket 'relud' at 'https://gcs-emulator:4443/storage/v1/b/relud' deleted.
```

Reply:
Thanks for the detailed steps! Testing this the first way is what hangs for me.
```bash
(venv) bdanforth ~/Projects/tecken (bug-1827507) $ STORAGE_EMULATOR_HOST=https://localhost:4443 venv/bin/python bin/gcs_cli.py create relud
```
If I wait long enough, I get a deadline exceeded error from the Google Cloud client library. The request to the emulator is timing out. Sure enough I have the docker host and VS Code listening on the same port when I check via `lsof` and friends.

Reply:
I don't consider this thread blocking, so I'm going to mark it as resolved for now to reduce noise. I reverted the default host port to 4443.